tema1/bio.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Honest blog</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <header style="background-color:olive;">
+    <h1>Lavinia's blog</h1>
+    <nav>
+        <a href="#" title="home">Home</a> |
+        <a href="#" title="despre">Despre mine</a> |
+        <a href="#" title="hobbys">Hobbys</a> |
+        <a href="#" title="places">Favorite places</a> |
+        <a href="#" title="links">Linkuri utile</a>
+    </nav>
+  </header>
+  <main style="background-color:wheat;"">
```

Comment:
Ai pus un set de ghilimele in plus :D 
```suggestion
  <main style="background-color:wheat;">
```